### PR TITLE
Add comments

### PR DESCRIPTION
**Pull request by Dbz**
Generate comments above the aliases so that the file is more human readable. Also add comments for the 1-off commands we add

### CODE REVIEW

**Comment by Dbz** [comment]
I need to update the README.md with this new functionality and the requirements

File: aliases.yaml [hunk 1]
```diff
@@ -77,8 +77,18 @@ cmds:
...
   - short: g
     cmd: get
+  - short: g
+    cmd: get
+    suffix:
+      short: g
+      cmd: " -o wide | grep"
   - short: ga
     cmd: "get --all-namespaces -o wide"
+  - short: ga
```
Comment (Dbz) [inline aliases.yaml:87]: TODO: remove this duplicate 2 lines

File: aliases.yaml [hunk 2]
```diff
@@ -77,8 +77,18 @@ cmds:
...
+    cmd: "get --all-namespaces -o wide"
+    suffix:
+      short: g
+      cmd: " -o wide | grep"
```
Comment (Dbz) [inline aliases.yaml:91]: We duplicate `-o wide`
```suggestion
      cmd: " | grep"
```

File: aliases.yaml (hunk 1 above)
Comment (Dbz) [reply aliases.yaml:87]: In retrospect. This might require some refactoring to make it all work. I'm not sure

Comment (patmessina) [reply aliases.yaml:87]: Yea, I think so. They end up with two different commands though. I think we would have to have a more complicated design for the yaml to make it work. Im not opposed to that. But this seems a little more explicit.

**Pull request merged**